cc_plugin_ugrid/checker_10.py

```diff
@@ -120,21 +121,69 @@ def check_node_coordinates_exist(self, ds):
             return None
         except AttributeError:
             # No node_dimensions attribute... there are larger issues
+            self.node_dim = None
```

Comment:
is it possible to move 

`self.node_dim = None` above, so it's set to None, only to be replaced by final condition?